.gitignore

```diff
@@ -77,3 +77,4 @@ sftp-config.json
 target/
 var/
 venv/
+backup/
```

Comment:
I think this is unnecessary for the project. Also, the files and paths should be in alphabetical order.